setup.py

```diff
@@ -12,7 +12,10 @@
     author="Algorand",
     author_email="pypiservice@algorand.com",
     python_requires=">=3.8",
-    install_requires=["py-algorand-sdk", "tabulate==0.8.9"],
+    install_requires=[
+        "py-algorand-sdk @ git+https://github.com/algorand/py-algorand-sdk@get-method-by-name",
```

Comment:
change back to PyPi `py-algorand-sdk` once the branch is merged and released

Reply:
Ben just merged the PR, shall we change the dependency?
https://github.com/algorand/py-algorand-sdk/pull/345

Reply:
Thx! I'm waiting on the release to PyPi before reverting completely and merging the PR.

Reply:
👍